app/lib/actions/katello/content_view_version/destroy.rb

```diff
@@ -4,15 +4,18 @@ module ContentViewVersion
       class Destroy < Actions::Base
         def plan(version, options = {})
           version.check_ready_to_destroy! unless options[:skip_environment_check]
+          destroy_env_content = !options.fetch(:skip_destroy_env_content, false)
+          repos = destroy_env_content ? version.repositories : version.archived_repositories
```

Comment:
@daviddavis @jlsherrill can one of you help with the logic of this. So when we want to delete a cv version why would we want to say "please delete only the archived repositories" vs all repositories belonging to that version? I am I thinking wrong?